.github/scripts/cygwin/build-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+PATH="$TOOLCHAIN_PATH/bin:$TOOLCHAIN_PATH/lib/gcc/aarch64-pc-cygwin/:$TOOLCHAIN_PATH/lib/gcc/$TARGET/$GCC_VERSION/:$PATH"
+
+echo "::group::Build Cygwin AArch64 tests"
+    pushd $ROOT_PATH/tests/cygwin
+        mkdir -p $TESTS_PATH
+        aarch64-pc-cygwin-gcc -o $TESTS_PATH/hello-cygwin.exe hello-cygwin.c
```

Comment:
Here, `$TARGET-gcc` can be used as well.

Reply:
Changed